package.json

```diff
@@ -10,6 +10,15 @@
   "license": "ISC",
   "dependencies": {
     "react": "^15.5.4",
+    "react-dom": "^15.5.4",
     "redux": "^3.6.0"
```

Comment:
Remove this dependency. We may use it in the future, but it is not needed right now.